### DISCUSSION TITLE
Parent Buffer launched with Subscriptions

### DISCUSSION THREAD

**Discussion by umop3plsdn** [trizen/pipe-viewer#84] [General]
When I launch pipe-viewer with my subscriptions is it possible to get back to the First screen without having to end the program and restart with -S after i've explored a few videos from one of the subscriptions?

**Comment by trizen** [comment]
Use `:r`.

**Comment by umop3plsdn** [comment]
Nice thanks I don't know how I missed that! on the topic of subs my list always says   0 videos  0 subs on every single subscription. Is this intended? I do notice if I just search a name it actually gives the results. It's not really a big deal I know how many vids and subs I was just wondering what the purpose was for the output of it in the first place if it was intended to be 0
